### ISSUE TITLE
Allow assignment of the same hotkey to multiple window titles

### ISSUE THREAD

**Issue by williamcpierce** [williamcpierce/Overview#89]
> Llilith Day — 2024-12-19
Hey WP - I'm not sure if this has been mentioned - but I'd like to create multiple copies of the same hotkey.  
Example: I run 4 accounts with 3 toons each. I put the 4 previews in the same spot. At the moment, cmd + q selects the > first preview for 1 toon, but I can't assign cmq + q for the other 2 toons on that same account.
https://discord.com/channels/1295309622445473865/1295309623074750571/1319307916263555175

> justalbertosss3 — 2025-01-19
so im trying to set up the same hotkey for different characters(they are on the same account) and it wont let me
is there a way to override this?
https://discord.com/channels/1292215822990970981/1295516316509147166/1330613807361556480

> OldGuyMonty/BitchPudding — 2025-02-01
mmmkay....another thought......I'm using hotkeys tied to my StreamDeck Mini to switch between Windows. This works great when on my main characters, BUT when I log out and go to an alt, the window name changes which invalidates the hotkey. Being able to tie hot keys to multiple windows would be helpful. Intended usage is to keep the same hotkey across all three characters on an account
Basically, 1 hotkey for 3 windows (Toon 1, 2, 3) Only 1 of which is active at any given time. Maybe make it so that that you have a hotkey tied to an account, which lists out the 3 toons/window names. If x is active then hotkey.
https://discord.com/channels/1292215822990970981/1295516316509147166/1335267233605091359
